treeherder/seta/analyze_failures.py

```diff
@@ -0,0 +1,101 @@
+import logging
+
+from treeherder.etl.seta import parse_testtype
+from treeherder.model import models
+from treeherder.seta.common import unique_key
+from treeherder.seta.models import JobPriority
+from treeherder.seta.seta import get_high_value_jobs
+from treeherder.seta.update_job_priority import update_job_priority_table
+from treeherder.webapp.api.jobs import JobsViewSet
+
+HEADERS = {
+    'Accept': 'application/json',
+    'User-Agent': 'treeherder-seta',
+}
+LOG = logging.getLogger(__name__)
+
+
+class AnalyzeFailures:
+    def __init__(self, **options):
+        self.dry_run = options['dry_run']
+
+    def run(self):
+        revisions_fixed_by_commit_plus_tagged_jobs = get_failures_fixed_by_commit()
```

Comment:
I know this is descriptive, but it is very long, I would prefer something much shorter "fixed_by_commit_jobs" or something like that.  If wlach doesn't mind the long name, I would be fine with it.